bin/main.dart

```diff
@@ -452,20 +456,15 @@ Future<Map<String, String>> _loadStrings(String lang, String dirPath) async {
   var localizedStrings = <String, String>{};
   try {
     final file = File('$dirPath/$lang.json');
-    if (!await file.exists()) {
-      return localizedStrings;
-    }
+    if (!await file.exists()) return localizedStrings;
```

Comment:
please return bracket, it is a rule from linter

Reply:
OK. But the guide says "Exception: When you have an if statement with no else clause and the whole if statement fits on one line, you can omit the braces if you prefer"

Reply:
ok, then let's keep it